shared-configs/jest-config/jest.config.cjs

```diff
@@ -0,0 +1,17 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+const path = require("path");
+
+module.exports = {
+    preset: "ts-jest",
+    testEnvironment: "jsdom",
+    testEnvironmentOptions: {
+        url: "https://localhost:8081/index.html"
+    },
+    collectCoverageFrom: ["src/**/*.ts"],
+    coverageReporters: [["lcov", { "projectRoot": path.join(__dirname, "../../") }]],
+    setupFilesAfterEnv: [path.join(__dirname, "setupCrypto.cjs")],
```

Comment:
Curious: Was this optional before on `common` and `node-extensions`?

Reply:
Crypto APIs aren't used in those libs today so it hasn't been needed